Userland/Libraries/LibJS/Runtime/Object.cpp

```diff
@@ -893,17 +893,19 @@ ThrowCompletionOr<Value> Object::internal_get(PropertyKey const& property_key, V
             return js_undefined();
 
         // c. Return ? parent.[[Get]](P, Receiver).
-        return parent->internal_get(property_key, receiver);
+        return parent->internal_get(property_key, receiver, nullptr, PropertyLookupPhase::PrototypeChain);
     }
 
     // 3. If IsDataDescriptor(desc) is true, return desc.[[Value]].
     if (descriptor->is_data_descriptor()) {
         // Non-standard: If the caller has requested cacheable metadata and the property is an own property, fill it in.
         if (cacheable_metadata && descriptor->property_offset.has_value() && shape().is_cacheable()) {
-            *cacheable_metadata = CacheablePropertyMetadata {
-                .type = CacheablePropertyMetadata::Type::OwnProperty,
-                .property_offset = descriptor->property_offset.value(),
-            };
+            if (phase == PropertyLookupPhase ::OwnProperty) {
```

Comment:
An accidental extra whitespace.

```suggestion
            if (phase == PropertyLookupPhase::OwnProperty) {
```